cachito/web/migrations/env.py

```diff
@@ -23,7 +24,7 @@
 # target_metadata = mymodel.Base.metadata
 from flask import current_app
 
-config.set_main_option("sqlalchemy.url", current_app.config.get("SQLALCHEMY_DATABASE_URI"))
+config.set_main_option("sqlalchemy.url", str(current_app.config.get("SQLALCHEMY_DATABASE_URI")))
```

Comment:
this is suspicious `str(None)`

Reply:
Oops, forgot to fix this in the morning. Let me know if the change looks sane

Reply:
why do we need `str()` there?

Reply:
@MartinBasti I originally added str() to fix the mypy error:
`cachito/web/migrations/env.py:26: error: Argument 2 to "set_main_option" of "Config" has incompatible type "Optional[Any]"; expected "str"  [arg-type]`
But direct indexing appears to resolve it without str() :+1: Thanks @chmeliik